scripts/codemod/modifyProp.js

```diff
@@ -0,0 +1,96 @@
+// @flow strict
+
+/**
+ * CODEMOD to MODIFY (DEPRECATE or  RENAME) GESTALT COMPONENT PROPS
+ * Ex. <Box size="" /> to <Box />
+ * Ex. <Box size="" /> to <Box renamedSize="" />
+ * Ex. <Dropdown.Item size="" /> to <Dropdown.Item renamedSize="" />
+ *
+ *
+ * OPTIONS:
+ * --action: 'deprecate' removes prop matching previousPropName, 'rename' replaces prop name matching previousPropName with nextPropName
+ * --componentName: component to which modify props
+ * --subcomponentName: component's subcomponent to which modify props
+ * --previousPropName: current prop name to be replaced
+ * --nextPropName: (only required with rename action) new prop name to replace with
+ *
+ *
+ * TO RUN THIS CODEMOD
+ * yarn run:codemod modifyProp  <folder/file path> --action=<value> --previousPropName=<value> --nextPropName=<value>
+ * E.g. yarn run:codemod modifyProp  ~/code/pinboard/webapp --action=rename --componentName=Box --previousPropName=size --nextCmpName=renamedSize
```

Comment:
The example path here is making some assumptions about the user's directory structure…something more like `~/path/to/your/code` is a bit more generic